### PR TITLE
add missing events and event properties

### PR DESCRIPTION
**Pull request by rkusa**
I went through all events, testing them, adding missing ones, and adding missing properties. I've ignored all events I couldn't find a way to trigger.

### CODE REVIEW

File: lua/methods/mission.lua [hunk 1]
```diff
@@ -5,17 +5,18 @@ local function exporter(object)
...
-  elseif(category == Object.Category.STATIC) then
+  elseif category == Object.Category.STATIC then
     return GRPC.exporters.static(object)
-  elseif(category == Object.Category.BASE) then
-    return GRPC.exporters.airbase(object)
-  elseif(category == Object.Category.SCENERY) then
+  elseif category == Object.Category.SCENERY then
     return GRPC.exporters.scenery(object)
-  elseif(category == Object.Category.Cargo) then
+  elseif category == Object.Category.Cargo then
```
Comment (rkusa) [inline lua/methods/mission.lua:19]: Removed the parenthesis for consistency with the other Lua code. But I'd also be fine to change everything to using parenthesis instead

Comment (rurounijones) [reply lua/methods/mission.lua:19]: Parethessislessness is fine :grin:

File: src/stream.rs [hunk 2]
```diff
@@ -133,6 +133,7 @@ async fn handle_event(state: &mut State, event: Event) -> Result<(), Error> {
                 Some(Initiator {
                     initiator: Some(initiator::Initiator::Unit(unit)),
                 }),
+            ..
```
Comment (rurounijones) [inline src/stream.rs:136]: Is this supposed to be here?

Comment (rkusa) [reply src/stream.rs:136]: Yes, the struct has now another property, something the compiler would complain about. This is the syntax for telling the compiler that we don't care about any other property of the struct 👍

Comment (rurounijones) [reply src/stream.rs:136]: Nice, Today I learned.

**Pull request merged**